### PR TITLE
Addnp

### PR DESCRIPTION
**Pull request by mynameissaraah**
New verified Non-profits were added 
Payment information for some of the non profits were updated

### CODE REVIEW

File: src/non-profits-data/index.ts [hunk 1]
```diff
@@ -134,6 +145,31 @@ const nonProfits: readonly NonProfit[] = [
         type: 'xrp',
         paymentID: 'rGeyCsqc6vKXuyTGF39WJxmTRemoV3c97h',
       },
+
```
Comment (uchibeke) [inline src/non-profits-data/index.ts:148]: Could you reference where this wallet addresses where gotten from?

File: src/non-profits-data/index.ts [hunk 2]
```diff
@@ -483,6 +519,170 @@ const nonProfits: readonly NonProfit[] = [
...
+    verifications: [
+      {
+        date: new Date('Oct 4, 2023'),
+        verifier: '@mynameissaraah',
+      },
+    ],
+    paymentMethods: [
+      {
+        type: 'Bank tranfer',
+        paymentID: 'Account No: 0019052855 - Sterling Bank Plc 0034611000 Diamond Bank Plc Account ',
```
Comment (uchibeke) [inline src/non-profits-data/index.ts:574]: Could you reference where this information was gotten from?

File: src/non-profits-data/index.ts [hunk 3]
```diff
@@ -483,6 +519,170 @@ const nonProfits: readonly NonProfit[] = [
...
+    isVerified: true,
+    verifications: [
+      {
+        date: new Date('Oct 4, 2023'),
+        verifier: '@mynameissaraah',
+      },
+    ],
+    paymentMethods: [
+      {
+        type: 'Mail',
```
Comment (uchibeke) [inline src/non-profits-data/index.ts:596]: For these new Payment methods, as part of your PR, could you update the readme and add a Types section in the format below? This ensures we have a standardized payment method list so people are not creating any method. Also if you check the API, notice you can get NP by payment type. So we need standard codes/names for the payment types we are adding and also need to document them.

```
### Payment Types
These are the existing payment types that support
- xrp
- interledger
- mobile-money
- airtime
- new-type 1
- new-type 2
- new-type3
```

Please include all existing and new methods to the readme.

**Review by uchibeke — CHANGES_REQUESTED** [review]

File: src/non-profits-data/index.ts (hunk 3 above)
Comment (mynameissaraah) [reply src/non-profits-data/index.ts:596]: ok noted

File: src/non-profits-data/index.ts (hunk 2 above)
Comment (mynameissaraah) [reply src/non-profits-data/index.ts:574]: Should I reference it as a comment?

Comment (uchibeke) [reply src/non-profits-data/index.ts:574]: Yes please. Thanks 🚀

Comment (mynameissaraah) [reply src/non-profits-data/index.ts:577]: Done

File: src/non-profits-data/index.ts (hunk 3 above)
Comment (mynameissaraah) [reply src/non-profits-data/index.ts:600]: @uchibeke I have made the requested changes

**Pull request closed without merging**